### ISSUE TITLE
Github Releases coinciding with nuget releases?

### ISSUE THREAD

**Issue by ADNewsom09** [toddams/RazorLight#300]
Can we get releases on here coinciding with NuGet releases, easier to see what's available+ diffs between versions? Plus don't have to see alpha-3 as the latest release.

**Comment by toddams** [comment]
That's a good point. We should create them

**Comment by jzabroski** [comment]
@ADNewsom09  What do you mean, alpha-3 is seen as the latest release?

**Comment by jzabroski** [comment]
Ah, you mean this page: https://github.com/toddams/RazorLight/releases

**Comment by ADNewsom09** [comment]
I think that has to do with version numbers, but yeah. Otherwise it is really difficult to tell what commit a nuget release is on to find diffs and hey debug issues.

**Comment by jzabroski** [comment]
So, that's why snupkg exists, though.  Whatever we upload to nuget.org as a nupkg now has a corresponding snupkg.

I'm fine doing extra paperwork, but if your user story is to debug issues, why do you need snupkg plus GitHub Releases?  Just wondering so I understand.  We probably just have different workflows, and I'd like to learn to see if your approach is better (one of the great things about FLOSS projects is learning how people do things).

**Comment by ADNewsom09** [comment]
Ah, debugging isn't the right word, following the dev process in git and seeing where it branches off the Nuget release and offering PRs for fixes would be more in-line. Thanks guys.

**Comment by jzabroski** [comment]
We love PRs and help.  Will do that.

**Comment by jzabroski** [comment]
@ADNewsom09 I don't think I did this right - @toddams had a 2.0.0 in "Draft" status that I pushed as Pre-release.  However, I think I should have updated the release title to 2.0.0-beta3 and incorporated the various changes.

What are your thoughts on fixing this mess?

My thinking is, the easiest thing to do is "ding dong and ditch" - just move on and start following best practices starting with 2.0.0-beta4.  I still haven't gotten through the whole backlog of bugs and I think fixing bugs is probably more important than maintaining this GitHub tab.

Thoughts?
